src/snappy_device_agents/data/limerick/user-data

```diff
@@ -0,0 +1,14 @@
+#cloud-config
+ssh_pwauth: True
+users:
+ - name: ubuntu
+   gecos: ubuntu
+   uid: 1000
+   shell: /bin/bash
+   lock_passwd: False
+   groups: [ adm, dialout, cdrom, floppy, sudo, audio, dip, video, plugdev, lxd, netdev, render ]
+   plain_text_passwd: 'ubuntu'
+chpasswd:
+  list: |
+    ubuntu:ubuntu
+  expire: True
```

Comment:
I think we should remove this `expire: True`, with this line inside of user-data a change of password will always be requested by the first boot and that's something we don't want.